recipes/sparse/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sparse" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "bb433d9adf16b0163436e3890bd1d14694d813697cacabd38434830f7901885e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >= 1.14
+    - scipy >= 0.19
+    - numba >= 0.37
+
+test:
+  imports:
+    - sparse
+
+about:
+  home: https://github.com/pydata/sparse
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.rst
+  summary: 'Sparse multi-dimensional arrays for the PyData ecosystem'
+
+  description: |
+    sparse implements sparse multidimensional arrays on top of Numpy
+    and scipy.sparse. It tries to mimic Numpy's ndarray API.
+  doc_url: https://sparse.pydata.org/en/0.3.0/
```

Comment:
Maybe drop the version number from the `doc_url`? I'm afraid it might not get updated when the recipe does since it's buried far down in the recipe.

Reply:
I could, but since this package is currently not mature I expect the information on there to get out of sync with what's in the recipe quite quickly, which is why I added a link to the release version.